unittest/test_module.py

```
@@ -2,31 +2,20 @@
 from dataclasses import asdict
+from datetime import datetime
 
 from aiohttp import ClientConnectorSSLError
 from botpy.errors import ServerError
 
 from src.core.exception import handle_exception, UnauthorizedError, ModuleRuntimeError
 from src.core.tools import png2jpg, decode_range
-from src.module.color_rand import load_colors, _colors, render_color_card, add_qrcode
-from src.platform.cp.atcoder import AtCoder
-from src.platform.cp.codeforces import Codeforces
+from src.module.color_rand import load_colors, _colors, add_qrcode
```

Ruff (0.8.2)

`src.module.color_rand.load_colors` imported but unused

Remove unused import

(F401)